### PR TITLE
raft: cleanup node

### PR DESCRIPTION
**Pull request by NingLin-P**
In tinykv, `RawNode` are used as the interface of raftgroup and `node` is redundant.

### CODE REVIEW

File: raft/rawnode.go [hunk 1]
```diff
@@ -27,6 +27,112 @@ var ErrStepLocalMsg = errors.New("raft: cannot step raft local message")
...
+	if r.RaftLog.unstable.snapshot != nil {
+		rd.Snapshot = *r.RaftLog.unstable.snapshot
+	}
+	rd.MustSync = MustSync(r.hardState(), prevHardSt, len(rd.Entries))
+	return rd
+}
+
+// MustSync returns true if the hard state and count of Raft entries indicate
+// that a synchronous write to persistent storage is required.
+func MustSync(st, prevst pb.HardState, entsnum int) bool {
```
Comment (Connor1996) [inline raft/rawnode.go]: let's remove it, TinyKV will sync everytime

File: kv/tikv/config/config.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ type Config struct {
 	RaftElectionTimeoutTicks    int
 	RaftMinElectionTimeoutTicks int
 	RaftMaxElectionTimeoutTicks int
-	RaftMaxSizePerMsg           uint64
+	RaftMaxEntsSize             uint64
```
Comment (Connor1996) [inline kv/tikv/config/config.go]: can we remove it?

File: raft/example_test.go [hunk 3]
```diff
@@ -1,47 +0,0 @@
...
-import (
-	pb "github.com/pingcap-incubator/tinykv/proto/pkg/eraftpb"
-)
-
-func applyToStore(ents []pb.Entry)    {}
-func sendMessages(msgs []pb.Message)  {}
-func saveStateToDisk(st pb.HardState) {}
-func saveToDisk(ents []pb.Entry)      {}
-
-func ExampleNode() {
```
Comment (Connor1996) [inline raft/example_test.go]: let's write the exmaple code to course material

Comment (NingLin-P) [reply raft/example_test.go]: OK, I left a TODO for now and will fill it later when writing the lab 2 doc.

File: kv/tikv/config/config.go (hunk 2 above)
Comment (NingLin-P) [reply kv/tikv/config/config.go]: I will remove it in a upcoming pr about `Storage` and `RaftLog`.

**Pull request merged**